contracts/pizzabonus/pizzabonus.cpp

```diff
@@ -0,0 +1,24 @@
+#include <eosio/eosio.hpp>
+#include "./pizzabonus.h"
+
+pizzabonus::pizzabonus(name self_contract, name user, eosio::datastream<const char*> ds) 
+: contract(self_contract, user, ds), ut(""_n, ""_n.value) {
```

Comment:
https://stackoverflow.com/questions/43306044/compiler-error-because-constructor-must-explicitly-initialize-the-member